lib/Dialect/Linalg/Transforms/LinalgTransforms.cpp

```diff
@@ -106,3 +111,38 @@ bool mlir::linalg::detail::isProducedByOpOfTypeImpl(
   }
   return false;
 }
+
+LogicalResult
+mlir::linalg::permuteGenericLinalgOp(PatternRewriter &rewriter, Operation *op,
+                                     ArrayRef<unsigned> permutation,
+                                     StringRef linalgMarker) {
```

Comment:
Can we `s/linalgMarker/transformationStateMarker/g` ?
I named it poorly the first time around.

Reply:
Done (globally).